core/controller/src/main/scala/org/apache/openwhisk/core/controller/Activations.scala

```diff
@@ -37,6 +38,7 @@ import org.apache.openwhisk.core.entity._
 import org.apache.openwhisk.http.ErrorResponse.terminate
 import org.apache.openwhisk.http.Messages
 import org.apache.openwhisk.core.database.UserContext
+import pureconfig.loadConfig
```

Comment:
I know some other configs are needing the additional import `import pureconfig.generic.auto._`, but I'm not sure the exact cases where this is required. It may come up in system test failures.